dependencies/mac/README.md

```diff
@@ -0,0 +1,14 @@
+# Mac
+
+Most of the Xcode project work has been done for you. Once you've set up the following sources in this dependencies folder, you can build RSDKv3 and the frameworks will compile beforehand.
+
+## Install dependencies
```

Comment:
Are .framework and manual zip downloads the best way to deal with dependencies on OSX? Something like homebrew or macports seems simpler but probably not the "Apple way" of doing things.